### PR TITLE
Toy Rubber Duck Mine

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

Adds a toy version of the rubber ducky mine! This toy version only flashes the person who steps on it, and is still consumed in the process. 

Crafted with a rubber ducky (which are limited in supply right now) and a flash by hand!

## Why It's Good For The Game

It's another quirky fun item for hijinks and pranks

## Testing Photographs and Procedure

https://github.com/BeeStation/BeeStation-Hornet/assets/9547572/d7715c60-b632-4dd6-b81b-46d2caefa864

## Changelog
:cl:
add: Adds the toy rubber duck mine, crafted by hand from a rubber ducky and a flash. You can probably guess what it does when you step on the mine from its components. 
/:cl:

### CODE REVIEW

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]
* It should return a toy duck once it's used. likely `new /obj/item/bikehorn/rubberducky(get_turf(loc))` in Destroy(), so that we won't lose our precious ducky.
* recipe seems too simple even though flash is difficult to get. There should be some cable or anything more.

**Comment by Rukofamicom** [comment]
Can do

**Comment by PowerfulBacon** [comment]
It should play the explosion sound to the person who steps on it as an extra prank to make them think they blew up

**Comment by Rukofamicom** [comment]
Even actual duck mines usually don't because for some reason deafness sets in first.

**Comment by PowerfulBacon** [comment]
It should do whatever normal rubber ducky mines do so that it pranks you into thinking you actually blew up

**Comment by Rukofamicom** [comment]
will eventually do still :tm:

**Comment by Rukofamicom** [comment]
Oh shit I forgot I had this to finish up.

**Comment by Rukofamicom** [comment]
image

Recipe now, and the mine drops a burnt out flash + rubber ducky

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -788,6 +788,16 @@
...
 	category = CAT_MISC
 
+/datum/crafting_recipe/flash_ducky
+	name = "Toy Rubber Duck Mine"
+	result = /obj/item/deployablemine/traitor/toy
+	time = 20
+	reqs = list(/obj/item/bikehorn/rubberducky = 1,
+				/obj/item/assembly/flash/handheld = 1,
+				/obj/item/stack/cable_coil = 2,
+				/obj/item/assembly/prox_sensor)
```
Comment (EvilDragonfiend) [inline code/datums/components/crafting/recipes.dm:798]:
```suggestion
				/obj/item/assembly/prox_sensor)
	blacklist = list(/obj/item/assembly/flash/handheld/strong)
```
It's rev flash. used no longer currently, but let's not it be consumed by it.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

File: code/datums/components/crafting/recipes.dm (hunk 1 above)
Comment (Rukofamicom) [reply code/datums/components/crafting/recipes.dm:798]: Good call just in case.

**Review by EvilDragonfiend — APPROVED** [review]

**Pull request merged**